scaleway/resource_apple_silicon_server.go

```diff
@@ -38,7 +38,8 @@ func resourceScalewayAppleSiliconServer() *schema.Resource {
 				Required:    true,
 				ForceNew:    true,
 				ValidateFunc: validation.StringInSlice([]string{
```

Comment:
Should we remove this entierly? Otherwise we would be force to update everytime a new type comes along.